### ISSUE TITLE
Support for Scala 2.12/Akka 2.4

### ISSUE THREAD

**Issue by kutchar** [zmanio/atmos#13]
Are there any plans to publish artifacts for Scala 2.12/Akka 2.4?

**Comment by rators** [comment]
This library is fantastic. I would like to use it for a work project but the lack of Scala 2.12 support is a major concern. @lpryor Would you be open to accepting volunteers to maintain this repo?

**Comment by devshorts** [comment]
bump?

**Comment by kutchar** [comment]
Seems like @lpryor is not maintaining this project anymore. Should we fork it?

**Comment by devshorts** [comment]
Yes I think we should. I can cross publish what exists here under the io.paradoxical maven namespace (which I am a member if) if we want but I'm not sure I'm willing to be a full time fixer/maintainer

**Comment by devshorts** [comment]
@kutchar I forked and pushed this to io.paradoxical including 2.12 support https://github.com/paradoxical-io/atmos

**Comment by kutchar** [comment]
@devshorts awesome! I'll use your fork going forward. BTW, I was going to say that the library is pretty stable and feature complete so I don't really anticipate it'd need much "maintenance" to require a full-time fixer/maintainer

**Comment by devshorts** [comment]
@kutchar yeah sounds good. And if it needs to be handed off again, well we'll do that :)

**Comment by rators** [comment]
@devshorts was pleasantly surprised to look back at this pull request and see that someone took the initiative to fork this. As @kutchar said, I'll work on your branch going forward as well. Thanks @devshorts, you da man!.
